src/test/java/net/sf/jsqlparser/statement/select/SelectTest.java

```diff
@@ -2500,6 +2500,12 @@ public void testMultiValueIn4() throws JSQLParserException {
         assertSqlCanBeParsedAndDeparsed(stmt);
     }
 
+    @Test
+    public void selectIsolationKeywordsAsAlias() throws JSQLParserException {
+        String stmt = "SELECT col FROM tbl cs";
```

Comment:
You do not test your **isolation** usage.

Reply:
There are other tests already testing isolation usage (for example, _testWithIsolation_).
This PR just allows "cs" to be used as a column name / alias, as otherwise parsing fails when it's used.